app/controllers/application_controller.rb

```diff
@@ -2165,10 +2165,10 @@ def visibility_box_edit
 
   def get_record_display_name(record)
```

Comment:
@h-kataria, found flash message which renders after deletion is not in ```miq_ae_class_controller.rb```, it is https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/application_controller/ci_processing.rb#L163